tensorflow_serving/model_servers/main.cc

```diff
@@ -128,6 +131,20 @@ ModelServerConfig BuildSingleModelConfig(
   return config;
 }
 
+ModelServerConfig BuildModelConfigFromFile(
+    const string& config_file_path) {
+
```

Comment:
Can you please follow the pattern used for ParsePlatformConfigMap() (in particular, using the tensorflow Env to read the file rather than open()). Perhaps you can create a utility function that both BuildModelConfigFromFile() and ParsePlatformConfigMap() can call into. Thanks.

(And accordingly please eliminate #include <fcntl.h>.)